Rakefile

```diff
@@ -43,6 +43,28 @@ namespace :build do
       xcodebuild('build -scheme "Example (tvOS)" -destination "platform=tvOS Simulator,name=Apple TV" -workspace Lottie.xcworkspace')
     end
   end
+
+  desc 'Builds an xcframework for all supported platforms'
+  task :xcframework do
+    sh 'rm -rf .build/archives'
+    xcodebuild('archive -workspace Lottie.xcworkspace -scheme "Lottie (iOS)" -destination generic/platform=iOS -archivePath ".build/archives/Lottie_iOS" SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES')
```

Comment:
Maybe we could factor out the `SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES` into a constant since they're the same on every xcodebuild invocation?